### PR TITLE
fix: tree-wide pragma once

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
This is less error-prone than ifndef/define include guards and (now) widely supported by all compilers we care about.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
No.

### Does this PR change default behavior?
No.

### CODE REVIEW

File: src/utilities/eicrecon/print_info.h [hunk 1]
```diff
@@ -49,4 +48,3 @@ void printJANAHeaderIMG() {
                  " YMMMM9 _dM_     _dMM_M_      \\M _dM_     _dMM_MMMMMMMM " << std::endl << std::endl;
 }
```
Comment (veprbl) [inline src/utilities/eicrecon/print_info.h:50]: Would be nice to not leave a trailing whitespace

**Review by veprbl — APPROVED** [review]
LGTM

File: src/utilities/eicrecon/print_info.h (hunk 1 above)
Comment (wdconinc) [reply src/utilities/eicrecon/print_info.h:50]: Yeah, but my sed fu only goes so far and I was going to have pre-commit do that... I'll take a look later.

Comment (wdconinc) [reply src/utilities/eicrecon/print_info.h:50]: Resolved by #464.

**Pull request merged**